mondrian/src/main/java/mondrian/rolap/SqlStatement.java

```diff
@@ -535,6 +559,7 @@ public Object get(ResultSet resultSet, int column) throws SQLException {
             case LONG:
                 return resultSet.getLong(column + 1);
             case DOUBLE:
+            case DECIMAL:
```

Comment:
I don't remember why we have both Accessors which retrieve resultset values *and* a get() method in TYPE.  But whatever the reason, shouldn't the DECIMAL type use getBigDecimal here?

Reply:
Probably.  This seems to be some fallback default that doesn't ever get used, bug we might as well be consistent.